editor/settings_config_dialog.cpp

```diff
@@ -199,6 +199,14 @@ void EditorSettingsDialog::_update_icons() {
 
 void EditorSettingsDialog::_update_shortcuts() {
 
+	Map<String, bool> collapsed;
+
+	if ( shortcuts->get_root() && shortcuts->get_root()->get_children() ) {
```

Comment:
There are extra space after `(` and before `)` of `if` which fail the clang-format check. Other than that, the change looks fine. Thanks! :)